spqlios/arithmetic/vec_znx_arithmetic_private.h

```diff
@@ -157,32 +157,32 @@ struct module_info_t {
 EXPORT uint64_t fft64_bytes_of_vec_znx_dft(const MODULE* module,  // N
                                            uint64_t size);
 
-EXPORT VEC_ZNX_DFT* fft64_new_vec_znx_dft(const MODULE* module,  // N
+EXPORT VEC_ZNX_DFT* new_vec_znx_dft(const MODULE* module,  // N
```

Comment:
These one need to be declared in the public .h

Reply:
already there, so deleted these lines